package.json

```diff
@@ -17,7 +17,7 @@
     "start-clean": "rm -rf $TMPDIR/react-* && watchman watch-del-all && npm run start:native -- --reset-cache",
     "link": "npm link @gooddollar/goodcontracts",
     "web:local": "npm run link && npm run web",
-    "build": "npm run lingui:compile && NODE_OPTIONS=\"--openssl-legacy-provider --max_old_space_size=7000\" vite build",
+    "build": "npm run lingui:compile && NODE_OPTIONS=\"--openssl-legacy-provider --max_old_space_size=12000\" vite build",
```

Comment:
@johnsmith-gooddollar @sirpy 
Explanation: for me it heaped out building, even at 8gb

Reply:
cant use more than 7G because of vercel limits.
you can try to add more packages to the "manualChunks" to reduce the size of the main chunk which is usually the cause for these oom errors.

Reply:
yes already fixed in a follow up commit

Reply:
I needed the 12g for local, so added a build-12